File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        
        int index = 0;
        int prev = Integer.MIN_VALUE;
        
        for(int i = 0; i < nums.length; i++)
        {
            if(nums[i] != prev)
            {
                nums[index] = nums[i];
                prev = nums[i];
                index++;
            }
        }
        
        return index;
    }
}

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:

        
#         cnt = 0
#         prevValue = -inf
        
#         for val in nums:
#             if val != prevValue:
#                 nums[cnt] = val
#                 prevValue = val
#                 cnt += 1
                
#         return cnt


        index = 1
    
        for i in range(1,len(nums)):
            if nums[i] != nums[i-1]:
                nums[index] = nums[i]
                index += 1
                
        return index